fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/RangeInference.java

```diff
@@ -469,6 +486,9 @@ public ValueDesc union(ValueDesc other) {
 
         @Override
         public ValueDesc intersect(ValueDesc other) {
+            if (other instanceof EmptyValue) {
+                return other.intersect(this);
```

Comment:
why not return EmptyValue?

Reply:
its final result indeed return EmptyValue. Bug for unify function purpose,  when RangeValue/DiscreteValue/UnknownValue interset with an EmptyValue, call EmptyValue.interset(this) => EmptyValue